service/src/main/java/bio/terra/cbas/dependencies/sam/SamService.java

```diff
@@ -19,6 +23,10 @@ private StatusApi getStatusApi() {
     return new StatusApi(samClient.getApiClient());
   }
 
+  private UsersApi samUsersApi(String accessToken) {
```

Comment:
small nit: to keep this method consistent with other methods as well as prefix it with a verb, maybe it can be renamed to `getUsersApi`?